docs-chef-io/content/inspec/resources/k8s_deployments.md

```diff
@@ -11,7 +11,7 @@ identifier = "inspec/resources/k8s/K8s Deployments"
 parent = "inspec/resources/k8s"
 +++
 
-Use the `k8s_deployments` Chef InSpec audit resource to test the configurations of all Deployments in a namespace.
+Use the `k8s_deployments` Chef InSpec audit resource to test the configurations of all deployments in a namespace.
```

Comment:
Why is Deployments changed to deployments?
Just like CronJob, Deployments is a type of Kubernetes resource.